student.rb

```diff
@@ -1,12 +1,20 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
+  attr_reader :classroom
+
+  def initialize(id, age, classroom = nil, name = 'Unknown', parent_permission: true)
     @classroom = classroom
+    add_student_classroom if clasroom
```

Comment:
- [ ]  There is no need to add this line. kindly remove it as  `add_student_classroom` is not exist.